activerecord/CHANGELOG.md

```diff
@@ -1,37 +1,52 @@
+*   Add adapter option disallowing foreign keys
+
+    This adds a new option to be added to `database.yml` which enables skipping
+    foreign key constraints usage even if the underlying database supports them.
+
+    Usage:
+    ```yaml
+    development:
+        <<: *default
+        database: db/development.sqlite3
+        foreign_keys: false
+    ```
+
+    *Paulo Barros*
+
 *   Add configurable deprecation warning for singular associations
 
     This adds a deprecation warning when using the plural name of a singular associations in `where`.
     It is possible to opt into the new more performant behavior with `config.active_record.allow_deprecated_singular_associations_name = false`
 
     *Adam Hess*
 
-*   Run transactional callbacks on the freshest instance to save a given 
```

Comment:
Would you restore the original one, not to remove the training spaces in this pull request ? These changes are not related to this pull request goal. You can open another one if necessary.

Reply:
I should have mentioned that removing trailing spaces may be considered as cosmetic changes.